workos/passwordless.py

```diff
@@ -1,61 +1,79 @@
 
 
 class Passwordless(PasswordlessModule):
     """Offers methods through the WorkOS Passwordless service."""
 
-    @validate_settings(PASSWORDLESS_MODULE)
-    def __init__(self):
-        pass
-
-    @property
-    def request_helper(self):
-        if not getattr(self, "_request_helper", None):
-            self._request_helper = RequestHelper()
-        return self._request_helper
+    _http_client: SyncHTTPClient
 
-    def create_session(self, session_options):
+    @validate_settings(PASSWORDLESS_MODULE)
+    def __init__(self, http_client: SyncHTTPClient):
+        self._http_client = http_client
+
+    def create_session(
+        self,
+        email: str,
+        type: PasswordlessSessionType,
+        redirect_uri: Optional[str] = None,
+        state: Optional[str] = None,
+        expires_in: Optional[int] = None,
+    ) -> PasswordlessSession:
         """Create a Passwordless Session.
 
         Args:
-            session_options (dict) - An session options object
-                session_options[email] (str): The email of the user to authenticate.
-                session_options[redirect_uri] (str): Optional parameter to
-                    specify the redirect endpoint which will handle the callback
-                    from WorkOS. Defaults to the default Redirect URI in the
-                    WorkOS dashboard.
-                session_options[state] (str): Optional parameter that the redirect
-                    URI received from WorkOS will contain. The state parameter
-                    can be used to encode arbitrary information to help
-                    restore application state between redirects.
-                session_options[type] (str): The type of Passwordless Session to
-                    create. Currently, the only supported value is 'MagicLink'.
-                session_options[expires_in] (int): The number of seconds the Passwordless Session should live before expiring.
-                    This value must be between 900 (15 minutes) and 86400 (24 hours), inclusive.
+            email (str): The email of the user to authenticate.
+            redirect_uri (str): Optional parameter to
+                specify the redirect endpoint which will handle the callback
+                from WorkOS. Defaults to the default Redirect URI in the
+                WorkOS dashboard.
+            state (str): Optional parameter that the redirect
+                URI received from WorkOS will contain. The state parameter
+                can be used to encode arbitrary information to help
+                restore application state between redirects.
+            type (str): The type of Passwordless Session to
+                create. Currently, the only supported value is 'MagicLink'.
+            expires_in (int): The number of seconds the Passwordless Session should live before expiring.
+                This value must be between 900 (15 minutes) and 86400 (24 hours), inclusive.
```

Comment:
Changes like these definitely need documenting, but fortunately the developer will be able to take their existing dict and just spread it on to the method call. For example:

4.x usage:
```
session_options = {
    ...
}

create_session(session_options)
```

5.x usage:
```
session_options = {
    ...
}

create_session(**session_options)
```

Reply:
... so this was like the API function we decided should accept a `dict`?

Reply:
I was under the impression we were standardizing on separate parameters like this change. Reason being, once we're on 3.12+, with have [PEP 692](https://peps.python.org/pep-0692/), we can type kwargs with a TypedDict. Also fewer existing methods use this "dict" parameter pattern.

Either way we landed, I'm realizing we probably should've logged that somewhere. Let me know if you think I should swap this back to one dict parameter.